contracts/native/version-control/src/queries.rs

```diff
@@ -71,23 +74,23 @@ pub fn handle_module_list_query(
     start_after: Option<ModuleInfo>,
     limit: Option<u8>,
     filter: Option<ModuleFilter>,
-    yanked: bool,
 ) -> StdResult<Binary> {
     let limit = limit.unwrap_or(DEFAULT_LIMIT).min(MAX_LIMIT) as usize;
 
+    let ModuleFilter {
+        provider: ref provider_filter,
+        name: ref name_filter,
+        version: version_filter,
+        yanked,
```

Comment:
`include_yanked` would be a more appropriate name.

Reply:
it's not including yanked, it's only for yanked modules.

Reply:
Making it so it includes yanked at the right positions of the map would be hard/ very expressive to do, hence you either page over the registered OR the yanked modules.
Imo that's fine, the ability to yank the module is already pretty advanced for a SC...